lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -378,13 +379,24 @@ class LotForm extends React.Component {
                             >
                                 <div className="panel-body">
                                     <div className="col-xs-12">
+
                                         {currentLot.id && currentLot.plat &&
                                             <Notes content_type="Lot" object_id={currentLot.id} parent_content_type="Plat" parent_object_id={currentLot.plat.id} />
                                         }
                                     </div>
                                 </div>
                             </div>
                         </div>
+                        <div className="clearfix" />
+                        {lots.id &&
```

Comment:
Should we say currentLot.id here? Since we aren't clearing the state anymore with the lots reducer we wont have a single lot in lots.